develop/tutorials/articles/340-data-upgrades-and-verifiers/01-creating-an-upgrade-process-for-your-application.markdown

```diff
@@ -390,7 +390,7 @@ Here are the target's required attributes:
 - `release.bundle.symbolic.name`: module's bundle symbolic name
 - `release.schema.version`: module's current schema version
 
-For example, the `com.liferay.comment.page.comments.web` module's [`PageCommentsPortlet` class](https://docs.liferay.com/portal/7.0/javadocs/modules/apps/collaboration/comment/com.liferay.comment.page.comments.web/com/liferay/comment/page/comments/web/internal/portlet/PageCommentsPortlet.html)
+For example, the `com.liferay.comment.page.comments.web` module's [`PageCommentsPortlet` class](https://github.com/liferay/liferay-portal/blob/7.0.2-ga3/modules/apps/collaboration/comment/comment-page-comments-web/src/main/java/com/liferay/comment/page/comments/web/internal/portlet/PageCommentsPortlet.java)
```

Comment:
@mwilliams2014 - FYI - I took care of this collab class reference as I already had foundation class references in here.